Gemfile

```diff
@@ -25,7 +25,7 @@ gem 'pry-rails', '~> 0.3.4'
 gem 'rails_12factor', '~> 0.0.3', group: :production
 gem 'rack-cors', '~> 0.4.0', require: 'rack/cors'
 gem 'simple_token_authentication', '~> 1.6.0'
-
+gem 'paranoia'
```

Comment:
You can read the github readme, or add [label](https://github.com/jgorset/label) to your project